docs/src/plotting.md

```diff
@@ -0,0 +1,153 @@
+# Confusion matrices
+
+```@docs
+Lighthouse.plot_confusion_matrix
+```
+
+```@setup 1
+using CairoMakie
+CairoMakie.activate!(type = "png")
+
+```
+
+```@example 1
+using Lighthouse: plot_confusion_matrix, plot_confusion_matrix!
+confusion = [0.1 0.2 0.3 0.4 0.0;
```

Comment:
Since (as a user) it's easy to accidentally invert the labels on the x and y axes, it might be nice if the example use-case started from labeled data, to demonstrate expected behavior; if so, could construct a confusion matrix via Lighthouse calls/labeled data.

Reply:
Good point! 
Can you detail how you would improve the example?

Reply:
something like
```
classes = ["red", "orange", "yellow", "green"]
ground_truth = [1, 1, 1, 2, 2, 2, 3, 3, 3, 4, 4, 4]
predicted_labels = [1, 1, 1, 1, 2, 2, 4, 4, 4, 4, 4, 1]
confusion = confusion_matrix(length(classes), zip(predicted_labels, ground_truth))
fig, ax, p = plot_confusion_matrix(confusion, classes, :Row)
```